src/renderer/MultiLevelCellRenderer.ts

```diff
@@ -125,6 +150,26 @@ export default class MultiLevelCellRenderer extends AAggregatedGroupRenderer<IMu
           }
         });
 
+        const overlapData = stacked ? hasLabelOverlaps(context, cols, d, imposer) : null;
+        if (!overlapData) {
+          n.classList.remove(cssClass('stack-label'));
+          if (toWait.length > 0) {
+            return <IAbortAblePromise<void>>abortAbleAll(toWait);
+          }
+          return null;
+        }
+        n.classList.add(cssClass('stack-label'));
+
+        const combinedLabel = overlapData.map((col) => col.label).join('	');
```

Comment:
```suggestion
        const combinedLabel = overlapData.map((col) => col.label).join(' '); // use "em space" (U+2003 &#8195)
```

Copied from http://emptycharacter.com/